qcodes/instrument/parameter.py

```diff
@@ -302,8 +302,10 @@ def __init__(self, name: str,
             and not getattr(self.get_raw,
                             '__qcodes_is_abstract_method__', False)
         )
+        self.gettable = False
         if implements_get_raw:
             self.get = self._wrap_get(self.get_raw)
+            self.gettable = True
```

Comment:
it's `gettable` as opposed to `is_gettable` for conciseness? other options would be `has_get` or `can_get` (not a fan of this one).

Reply:
I have no strong oppinion on this any name if fine with me

Reply:
then we leave as is.